src/js/media-time-display.js

```diff
@@ -157,31 +149,104 @@ class MediaTimeDisplay extends MediaTextDisplay {
     this.tabIndex = -1;
   }
 
+  // Own props
+
+  /**
+   * Whether to show the remaining time
+   * @returns {boolean}
+   */
+  get remaining() {
+    return this.hasAttribute(Attributes.REMAINING);
+  }
+
+  /**
+   * Whether to show the remaining time
+   * @param {boolean} show
+   */
+  set remaining(show) {
+    if (show && !this.hasAttribute(Attributes.REMAINING)) {
+      this.setAttribute(Attributes.REMAINING, '');
+    } else if (!show && this.hasAttribute(Attributes.REMAINING)) {
+      this.removeAttribute(Attributes.REMAINING);
+    }
+    this.update();
```

Comment:
```suggestion
this.toggleAttribute(Attributes.REMAINING, Boolean(show));
```

I don't think it's really worth the extra code for the attribute exist check. what do you think?

`this.update();` should not be required I think because the attributeChangedCallback will be called and there it should have the `this.update();` call

Reply:
> don't think it's really worth the extra code for the attribute exist check. what do you think?

I was wondering the same thing! this is cleaner.

I think you're right about the update() it's going to run twice